Go/Go.sublime-syntax

```diff
@@ -416,44 +432,68 @@ contexts:
 
   match-floats:
     # Integer, no fraction, exponent
-    - match: \d+({{exponent}})\d+
+    - match: '{{ddigits}}{{exponent}}{{ddigits}}'
       scope: constant.numeric.float.go
-      captures:
-        1: punctuation.separator.exponent.go
     # Integer, fraction, optional exponent
-    - match: \d+(\.)\d+(?:({{exponent}})\d+)?
+    - match: '{{ddigits}}(\.){{ddigits}}(?:{{exponent}}{{ddigits}})?'
       scope: constant.numeric.float.go
       captures:
         1: punctuation.separator.decimal.go
-        2: punctuation.separator.exponent.go
     # Dot without fraction
     - match: \d+\.(?:{{exponent}}\d+)?
       scope: invalid.deprecated.go
     # Dot without integer
     - match: \.\d+(?:{{exponent}}\d+)?
       scope: invalid.deprecated.go
+    # Hex float literal, no fraction
+    - match: '(0[xX]){{hdigits}}[pP][+-]?{{ddigits}}(i)?'
+      scope: constant.numeric.float.go
+      captures:
+        1: punctuation.definition.numeric.hexadecimal.go
+        2: storage.type.numeric.go
+    # Hex float literal, with fraction
+    - match: '(0[xX]){{ohdigits}}(\.){{ohdigits}}[pP][+-]?{{ddigits}}(i)?'
+      scope: constant.numeric.float.go
+      captures:
+        1: punctuation.definition.numeric.hexadecimal.go
+        2: punctuation.separator.decimal.go
+        3: storage.type.numeric.go
 
   match-integers:
     - include: match-octal-integer
     - include: match-hex-integer
+    - include: match-binary-integer
     - include: match-decimal-integer
 
   match-octal-integer:
-    - match: (0)[0-7]+(?=\D)
+    - match: (0){{odigits}}(?=\D)
       scope: constant.numeric.octal.go
       captures:
         1: punctuation.definition.numeric.octal.go
     - match: 0[0-7]*[8-9]+
       scope: invalid.illegal.go
+    - match: (0[oO]){{odigits}}(i)?
+      scope: constant.numeric.octal.go
```

Comment:
Is this a `constant.numeric.integer.octal.go`?

Reply:
Resolved in 7500bd7.